spec/actions/bump_version_update_changelog_create_pr_action_spec.rb

```diff
@@ -148,11 +148,50 @@
         editor: editor
       )
     end
+
+    it 'adds important label to title and body' do
+      allow(FastlaneCore::UI).to receive(:interactive?).and_return(false)
+      allow(FastlaneCore::UI).to receive(:input).with('New version number: ').and_return(new_version)
+      allow(FastlaneCore::UI).to receive(:confirm).with(anything).and_return(false)
+      allow(File).to receive(:read).with(mock_changelog_latest_path).and_return(edited_changelog)
+      allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:validate_local_config_status_for_bump)
+        .with('release/1.13.0', mock_github_pr_token)
+      allow(Fastlane::Helper::VersioningHelper).to receive(:auto_generate_changelog)
+        .with(mock_repo_name, mock_github_token, 3)
+        .and_return(auto_generated_changelog)
+      allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:edit_changelog)
+        .with(auto_generated_changelog, mock_changelog_latest_path, editor)
+      allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:create_new_branch_and_checkout)
+        .with(new_branch_name)
+      allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:replace_version_number)
+        .with(current_version, new_version, ['./test_file.sh', './test_file2.rb'], ['./test_file3.kt', './test_file4.swift'])
+      allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:attach_changelog_to_master)
+        .with(new_version, mock_changelog_latest_path, mock_changelog_path)
+      allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:commmit_changes_and_push_current_branch)
+        .with("Version bump for #{new_version}")
```

Comment:
Hmm wonder if we should try to move all this to some function or something... But can be done in another PR though